[BUG] Parser/Linter error `function randomstr wants argument count 1 but provides 1`

Looks like a false alarm with the following error report?

```
🔥 [ERROR] function randomstr wants argument count 1 but provides 1 (function/arguments)
polyfill-service/fastly/vcl/fastly-boilerplate-begin.vcl at line 4, position 54
 3|    if (!req.http.X-Request-Id) {
 4|        set req.http.X-Request-Id = digest.hash_sha256(now randomstr(64) req.http.host req.url req.http.Fastly-Client-IP server.identity);
                                                              ^^^^^^^^^
 5|    }
See document in detail: https://developer.fastly.com/reference/vcl/functions/randomness/randomstr/
```

--- a/__generator__/builtin.yml
+++ b/__generator__/builtin.yml
@@ -793,6 +793,7 @@ randomstr:
   reference: "https://developer.fastly.com/reference/vcl/functions/randomness/randomstr/"
   on: [RECV, HASH, HIT, MISS, PASS, FETCH, ERROR, DELIVER, LOG]
   arguments:
+    - [INTEGER]
     - [INTEGER, STRING]
   return: STRING
 

--- a/context/builtin.go
+++ b/context/builtin.go
@@ -1190,6 +1190,7 @@ func builtinFunctions() Functions {
 			Value: &BuiltinFunction{
 				Return: types.StringType,
 				Arguments: [][]types.Type{
+					[]types.Type{types.IntegerType},
 					[]types.Type{types.IntegerType, types.StringType},
 				},
 				Scopes:    RECV | HASH | HIT | MISS | PASS | FETCH | ERROR | DELIVER | LOG,

--- a/context/predefined.go
+++ b/context/predefined.go
@@ -1399,7 +1399,7 @@ func predefinedVariables() Variables {
 							Items: map[string]*Object{},
 							Value: &Accessor{
 								Get:       types.StringType,
-								Set:       types.NeverType,
+								Set:       types.StringType,
 								Unset:     false,
 								Scopes:    RECV | ERROR | DELIVER | LOG,
 								Reference: "https://developer.fastly.com/reference/vcl/variables/client-connection/client-socket-congestion-algorithm/",
@@ -1409,7 +1409,7 @@ func predefinedVariables() Variables {
 							Items: map[string]*Object{},
 							Value: &Accessor{
 								Get:       types.IntegerType,
-								Set:       types.NeverType,
+								Set:       types.IntegerType,
 								Unset:     false,
 								Scopes:    RECV | ERROR | FETCH | DELIVER | LOG,
 								Reference: "https://developer.fastly.com/reference/vcl/variables/client-connection/client-socket-cwnd/",

